### PR TITLE
cli: add debug raft-log command

### PR DESCRIPTION
**Pull request by bdarnell**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4970)

### CODE REVIEW

File: cli/debug.go [hunk 1]
```diff
@@ -119,9 +124,73 @@ func runDebugRangeDescriptors(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var debugRaftLogCmd = &cobra.Command{
+	Use:   "raft-log [directory] [range id]",
+	Short: "print the raft log for a range",
+	Long: `
+  Prints all log entries in a store for the given range.
```
Comment (petermattis) [inline cli/debug.go]: `Long` help text is not supposed to be indented. Looks like this is done incorrectly in `debug.go`. Mind fixing while you're here?

File: cli/debug.go [hunk 2]
```diff
@@ -119,9 +124,73 @@ func runDebugRangeDescriptors(cmd *cobra.Command, args []string) error {
...
+	value := roachpb.Value{
+		RawBytes: meta.RawBytes,
+	}
+	var ent raftpb.Entry
+	if err := value.GetProto(&ent); err != nil {
+		return false, err
+	}
+	if len(ent.Data) > 0 {
+		var cmd roachpb.RaftCommand
+		if err := cmd.Unmarshal(ent.Data[9:]); err != nil {
```
Comment (petermattis) [inline cli/debug.go]: Where is that `9` coming from?

**Comment by petermattis** [comment]
LGTM

**Comment by spencerkimball** [comment]
Lgtm

File: cli/debug.go (hunk 2 above)
Comment (bdarnell) [inline cli/debug.go]: From `storage.raftCommandIDLen`. I'll export `storage.decodeRaftCommand` and use it here instead.

**Pull request merged**